### PR TITLE
add key wrapper for Identity struct

### PR DESCRIPTION
**Pull request by pxp928**
Signed-off-by: pxp928 <parth.psu@gmail.com>

created a key wrapper to capture specific pub key information that can later be used downstream.

### CODE REVIEW

File: pkg/ingestor/key/key.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package key
+
+type KeyType string
+
+type KeyScheme string
+
+/* const (
```
Comment (mlieberman85) [inline pkg/ingestor/key/key.go:22]: Is this commented section supposed to be here?

Comment (pxp928) [reply pkg/ingestor/key/key.go:22]: I put that as a reference but will be used by the verifier implementation to specify `KeyType` and `KeyScheme` into the `Identity` struct

Comment (mlieberman85) [reply pkg/ingestor/key/key.go:22]: Should we push in-toto to make this public? @SantiagoTorres ?

Comment (pxp928) [reply pkg/ingestor/key/key.go:22]: I will remove this comment for now.

**Comment by pxp928** [comment]
closing this as it will be captured in other PRs

**Pull request closed without merging**